ember-welcome-page/package.json

```diff
@@ -91,7 +92,7 @@
       "./components/welcome-page.js": "./dist/_app_/components/welcome-page.js"
     },
     "public-assets": {
-      "./src/images/construction.png": "/ember-welcome-page/images/construction.png"
+      "./images/construction.png": "/ember-welcome-page/images/construction.png"
```

Comment:
To learn more about `public-assets`, see https://github.com/embroider-build/embroider/blob/v1.8.3/SPEC.md#assets